src/result.ts

```diff
@@ -122,15 +123,15 @@ type Partitioned<E, A> = { errs: E[]; oks: A[] };
  * and one for the list of `Ok`s
  */
 const partition = <E, A>(results: Result<E, A>[]): Partitioned<E, A> =>
-    results.reduce(({ errs, oks }: Partitioned<E, A>, result) =>
-        either<E, A, Partitioned<E, A>>(
-            err => ({ errs: [ ...errs, err ], oks }),
-            ok => ({ errs, oks: [ ...oks, ok ] }),
-        )(result),
-        { errs: [], oks: [] },
+    results.reduce(
+        ({ errs, oks }: Partitioned<E, A>, result) =>
+            either<E, A, Partitioned<E, A>>(
+                (err) => ({ errs: [...errs, err], oks }),
+                (ok) => ({ errs, oks: [...oks, ok] })
+            )(result),
+        { errs: [], oks: [] }
```

Comment:
A lot of the diff on this PR appears to be ESLint/Prettier refactors? Are we able to set up ESLint/Prettier on this repo with `@guardian/eslint-config-typescript` in a separate PR and make those changes there? Just to keep the changes focused on the problem we're solving in this PR 🙂

Reply:
yeah ok sure, that's just what happened when I hit save :)

Reply:
oh blimey yeah, sorry, I didn't check the diff

Reply:
will turn this into a draft, merge #37 into this when that merges then reopen this

Reply:
done